readme.md

```diff
@@ -48,7 +48,7 @@ Milliseconds to humanize.
 
 #### options
 
-##### secDecimalDigits
+##### sec-decimal-digits
```

Comment:
I guess we can use `dec-digits` ? `secDecimalDigits` is great for the API,
but on the command line, typing something that long, well, it takes longer,
is all!